docs/spl-docs/evaluation-functions/comparison-conditional-functions.md

```diff
@@ -0,0 +1,177 @@
+# Comparison and Conditional functions
+
+_This list includes functions that can be used to return values based on conditions or compare different values._
+
+
+## case(\<condition\>, \<value\>, ...)
+This function takes in pairs of condition and value and returns the value corresponding to the first condition that evaluated to `true`.
```

Comment:
nit: these parameters are not clear. Do you mean:
```
case(\<condition1\>, \<value1\>, \<condition2\>, \<value2\>, ...)
```

Same for `validate()`

Reply:
> ... takes in pairs of condition and value and returns ...

nit: Something like this will be a little better:
```
... takes in pairs of conditions and values and returns ...
```

Same comment for `validate()`